src/components/carousel/Thumbnail.tsx

```diff
@@ -0,0 +1,37 @@
+import React, { FC, PropsWithChildren } from 'react';
+
+import AspectRatio from '../aspectRatio';
+
+interface Props {
+  onClick: () => void;
+  isCurrent: boolean;
+  thumbnailIndex: number;
+  totalThumbnails: number;
+}
+
+const Thumbnail: FC<PropsWithChildren<Props>> = ({
+  onClick,
+  isCurrent,
+  thumbnailIndex,
+  totalThumbnails,
+  children,
+}) => {
+  return (
+    <AspectRatio
+      ratio={4 / 3}
+      onClick={onClick}
+      width="133px"
```

Comment:
because it's a one-off value. I understood that we only introduce tokens for values that are reused in other places. Or am I mistaken?

Reply:
I meant more to use an existing one that's close instead of using a random value. We have `3xl | 8rem | 128px` which is probably close enough. (As long as there is no specific reason to use the hardcoded number)

Reply:
discussed with Enrique. He is fine with using 3xl 👍